resource/lvm/lv/lv.go

```diff
@@ -0,0 +1,128 @@
+package lv
+
+import (
+	"fmt"
+	"github.com/asteris-llc/converge/load/registry"
+	"github.com/asteris-llc/converge/resource"
+	"github.com/asteris-llc/converge/resource/lvm/lowlevel"
+	"github.com/pkg/errors"
+)
+
+type resourceLV struct {
+	group      string
+	name       string
+	size       *lowlevel.LvmSize
+	lvm        lowlevel.LVM
+	needCreate bool
+	devicePath string
+}
+
+// Status is a resource.Status extended by DevicePath of created volume
+type Status struct {
+	resource.Status
+	DevicePath string
+}
+
+func (r *resourceLV) Check(resource.Renderer) (resource.TaskStatus, error) {
+	status := &Status{}
+
+	// Check for LVM prerequizites
+	if err := r.lvm.Check(); err != nil {
+		return nil, errors.Wrap(err, "lvm.lv")
+	}
+
+	ok, err := r.checkVG()
+	if err != nil {
+		return nil, err
+	}
+
+	if ok {
+		lvs, err := r.lvm.QueryLogicalVolumes(r.group)
+		if err != nil {
+			return nil, err
+		}
+
+		_, ok = lvs[r.name]
+		r.needCreate = !ok
+	} else {
+		status.Output = append(status.Output, fmt.Sprintf("group %s not exist, assume that it will be created"))
```

Comment:
This outputs as:
```
root/lvm.lv.lv-test:
 Messages:
  group %!s(MISSING) not exist, assume that it will be created
 Has Changes: yes
 Changes:
  test: "<not exists>" => "created /dev/mapper/test-test"
```

Looks like the group name is missing from the `Sprintf` statement